corehq/apps/es/users.py

```diff
@@ -261,11 +267,14 @@ def _missing_user_data_value(property_name):
     A user_data property exists but has an empty string value.
     """
     return queries.nested(
-        'user_data_es',
+        "user_data_es",
         filters.AND(
```

Comment:
nit: this `AND` is no longer necessary, as there's a single clause inside it.  Though I think `_missing_user_data_value` and `_missing_user_data_property` would probably be simpler if combined, especially once a `domain` filter is applied.

Reply:
Removed `AND`